model/src/main/proto/javaclasses/exlibris/q/projections.proto

```diff
@@ -86,4 +106,228 @@ message HomePage {
     repeated Book book = 1;
     repeated Category category = 2;
 }
+
 // *** This is projections for UI role librarian ***
+
+// View for adding book.
+//
+// View consist of fields for writing book details.
+//
+message AddBookView {
+
+    //List of book categories.
+    repeated Category categories = 1;
+}
+
+// View for editing book.
+//
+// View consist of fields for edit book details.
+//
+message UpdateBookView {
+
+    // URL for book cover.
+    spine.net.Url cover_url = 1;
+
+    // Value that generated from decimal ISBN.
+    Isbn62 isbn = 2;
+
+    // Book’s title.
+    BookTitle title = 3;
+
+    // Full author’s name.
+    repeated Author authors = 4;
+
+    // Year of publication of the book.
+    Year year = 5;
+
+    // Categories of book.
+    repeated Category categories = 6;
+
+    // Full book’s description.
+    BookDescription description = 7;
+}
+
+// View for appending inventory.
+//
+// The second step of wizard of adding book items.
```

Comment:
second step of adding book instead of book items
or the only step of adding book items